automatic/opera/tools/chocolateyInstall.ps1

```diff
@@ -4,8 +4,9 @@ $toolsPath = (Split-Path -Parent $MyInvocation.MyCommand.Definition)
 
 $pp = Get-PackageParameters
 
-$parameters += if ($pp.NoDesktopShortcut)     { " /desktopshortcut 0"; Write-Host "Desktop shortcut won't be created" }
-$parameters += if ($pp.NoTaskbarShortcut)     { " /pintotaskbar 0"; Write-Host "Opera won't be pinned to taskbar" }
+$parameters += if ($pp.NoDesktopShortcut)     { " /desktopshortcut=0"; Write-Host "Desktop shortcut won't be created" }
+$parameters += if ($pp.NoTaskbarShortcut)     { " /pintotaskbar=0"; Write-Host "Opera won't be pinned to taskbar" }
+$parameters += if ($pp.InstallAllUsers)     { " /allusers=1"; Write-Host "Install Opera for all users, and not only the current one." }
```

Comment:
I am thinking that perhaps having this as the default would be most appropriate, and rather have a switch to only install for current user instead.

Thought?
/cc @majkinetor @gep13  @ferventcoder

Reply:
Agree.

Reply:
I'm going to test what this change in default behaviour will actually do. Depending on how the installer has been designed, there's a risk of this change resulting in an additional Opera installation for updating users.

Reply:
* This test was conducted by using the Chocolatey public repository for the single-user installation, and then my private repository for the all-users upgrade (the exact changes proposed here have already been incorporated internally).
* I started by using Revo Uninstaller to create the cleanest possible state for this test.

`choco install opera --version 62.0.3331.72` installed to `%LOCALAPPDATA%\Programs\Opera`

`choco upgrade opera --source="'$MYSOURCE'" --params /InstallAllUsers` (terminal output acknowledged the use of the /InstallAllUsers argument during the upgrade) updated the installation located in `%LOCALAPPDATA%\Programs\Opera` and left `Program Files` untouched.

So it looks like changing the default is safe.

Reply:
* This test was conducted by using the Chocolatey public repository for the single-user installation, and then my private repository for the all-users upgrade (the exact changes proposed here have already been incorporated internally).
* I started by using Revo Uninstaller to create the cleanest possible state for this test.

`choco install opera --version 62.0.3331.72` installed to `%LOCALAPPDATA%\Programs\Opera`

`choco upgrade opera --source="'$MYSOURCE'" --params /InstallAllUsers` (terminal output acknowledged the use of the /InstallAllUsers argument during the upgrade) updated the installation located in `%LOCALAPPDATA%\Programs\Opera` and left `C:\Program Files` untouched.

So it looks like changing the default is safe.

Reply:
That is great news, in that case would you mind updating the PR to have have all users set by default?

Thanks for testing it out.